newIDE/app/src/Export/ExportDialog/index.js

```diff
@@ -124,7 +124,7 @@ const ExportDialog = ({
         exporter.key !== 'onlinewebexport' && (
           <FlatButton
             key="builds"
-            label={<Trans>See all my builds</Trans>}
+            label={<Trans>See my game builds</Trans>}
```

Comment:
`See this game builds`?

to be very clear it's only this game being currently edited.